recipes/source.rb

```diff
@@ -1,6 +1,6 @@
 package 'git-core'
 
-git '/usr/src/exabgp' do
+git "#{node['exabgp']['src_path']}" do
```

Comment:
quotes not needed I believe?

Reply:
i think it's needed otherwise it acts like a comment